src/qutip_qip/device/circuitqed.py

```diff
@@ -63,6 +62,42 @@ class SCQubitsModel(Model):
     """
     The physical model for superconducting-qubit model processor
     (:obj:`.SCQubits`) with fixed frequency.
+    Each qubit is simulated by a multi-level Duffing model,
```

Comment:
Can we add a reference or link for the exact form to the Duffing model? By the way, always better to have somewhere the full Hamiltonian also with the cross-resonance.

Reply:
Good point, I'll add a reference. Isn't the full Hamiltonians are [below in the docstring](https://github.com/BoxiLi/qutip-qip/blob/f35525fe390746632f750158d16600ce9ea04840/src/qutip_qip/device/circuitqed.py#L80-L100)? Or are you expecting more details?